### PR TITLE
DOC: Fix wrong blockquotes

### PR DESCRIPTION
**Pull request by takanori-pskq**
Same as #16894.

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]

File: doc/source/user/c-info.how-to-extend.rst [hunk 1]
```diff
@@ -408,7 +404,6 @@ writeable). The syntax is
 
         Alternatively, the bit-width names can be used as supported on the
         platform. For example:
-
             :c:data:`NPY_INT8`, :c:data:`NPY_INT16`, :c:data:`NPY_INT32`,
```
Comment (mattip) [inline doc/source/user/c-info.how-to-extend.rst:407]: This one and the one on line 401 are needed

File: doc/source/user/c-info.how-to-extend.rst [hunk 2]
```diff
@@ -394,11 +392,9 @@ writeable). The syntax is
         to the requirements flag.
 
     *typenum*
-
         One of the enumerated types or :c:data:`NPY_NOTYPE` if the data-type
         should be determined from the object itself. The C-based names
         can be used:
-
             :c:data:`NPY_BOOL`, :c:data:`NPY_BYTE`, :c:data:`NPY_UBYTE`,
```
Comment (eric-wieser) [inline doc/source/user/c-info.how-to-extend.rst:398]:
```suggestion

            :c:data:`NPY_BOOL`, :c:data:`NPY_BYTE`, :c:data:`NPY_UBYTE`,
```

File: doc/source/user/c-info.how-to-extend.rst (hunk 1 above)
Comment (eric-wieser) [reply doc/source/user/c-info.how-to-extend.rst:407]:
```suggestion

            :c:data:`NPY_INT8`, :c:data:`NPY_INT16`, :c:data:`NPY_INT32`,
```

**Comment by mattip** [comment]
Thanks @takanori-pskq

**Pull request merged**